### PR TITLE
[next] Fixes #1547: Ported HelpPopoverButton from Gatsby to Next.js

### LINKED ISSUE #1547: [next] Port HelpPopoverButton component to NextJS
https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/gatsby/src/components/MyFeedsPage/HelpPopoverButton.jsx
**What would you like to be added**:
Port HelpPopoverButton component to NextJS using TS
**Why would you like this to be added**:
Porting reason

### PR DESCRIPTION
**Pull request by nilan01**
## Issue This PR Addresses 
Closes #1547 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI
- [x] **Porting NextJS**

## Description

Porting HelpPopoverButton from Gatsby to NextJS. Process was pretty straight forward. To test, I also rendered this on `index.tsx`.

Steps to test:
1) Open src/frontend/next/index.tsx
2) Add this to import statements: `import HelpPopoverButton from '../components/MyFeedsPage/HelpPopoverButton';`
3) Render component `<HelpPopoverButton />` in `index.tsx`

`package-lock.json` was modified as I had to run `npm install --save material-ui-popup-state`.

image

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/frontend/next/src/components/MyFeedsPage/HelpPopoverButton.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Box, IconButton, Popover, Typography } from '@material-ui/core';
```
Comment (humphd) [inline src/frontend/next/src/components/MyFeedsPage/HelpPopoverButton.tsx:1]: I don't think we should have a `MyFeedsPage` directory.  With next.js, we have directory for pages, and another for components.  The myfeeds page can be separated out.  Let's move this directly into the `components/` dir.

**Review by humphd — CHANGES_REQUESTED** [review]
Looks good.  One issue to deal with.

File: src/frontend/next/src/components/MyFeedsPage/HelpPopoverButton.tsx (hunk 1 above)
Comment (nilan01) [reply src/frontend/next/src/components/MyFeedsPage/HelpPopoverButton.tsx:1]: Hi @humphd, thank you for your feedback! I've removed `MyFeedsPage` and saved the `HelpPopoverButton` into `components/` dir.

**Review by chrispinkney — APPROVED** [review]
LGTM! Well done 😃

**Review by humphd — APPROVED** [review]
Nice work, @nilan01.  Welcome to Telescope!

**Pull request merged**